docs/design/consensus.md

```diff
@@ -1,24 +1,30 @@
 ## Consensus
 
-The consensus protocol in FTLib acts as a shadow precondition for any collective communication operations. Any changes from the consensus protocol will reset the initialization flag of FTLib to `False`, deterring any communication operations after the rebuild procedure returns success.
+The consensus protocol in FTLib acts as a shadow precondition for any collective communication operations. Any changes from the consensus protocol will reset the `_initialized` flag of FTLib to `False`, deterring any communication operations after the rebuild procedure returns success. Under passive or active mode, such change takes place immediately or eventually (when FTLib check the consensus next time).
```

Comment:
Under passive or active mode -> depending on the choice of passive or active mode